manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -224,7 +224,7 @@ spec:
           timeoutSeconds: 1
         securityContext:
           privileged: false
-        {{- if not (eq .Values.platform "openshift") }}
+        {{- if (eq (coalesce .Values.platform .Values.global.platform) "openshift") }}
```

Comment:
Please don't reference both here - this chart already collapses globals, so you don't need to do a manual coalesce here as well.

Either reference `Values.platform` directly or reference `Values.global.platform` directly - not both.

Since platform _is_ a global, cross-chart constant value, like the existing global values here, I have a slight preference for that, but either way - we should only reference one.